gosyntect.go

```diff
@@ -90,6 +118,66 @@ type Client struct {
 	syntectServer string
 }
 
+// sharedClient is a shared client aross all HighlightCSSTable requests
+// so connections to the server can be reused.
+var sharedClient = &http.Client{Transport: &nethttp.Transport{}}
```

Comment:
I added a package-level client so connections can be re-used. When we create a client for every request, we need to open a new connection every time.